tools/packaging/common/istio.service

```diff
@@ -4,6 +4,7 @@ Documentation=http://istio.io/
 
 [Service]
 ExecStart=/usr/local/bin/istio-start.sh
+ExecStopPost=/usr/local/bin/istio-start.sh clean
```

Comment:
Nit naming: istio-start that exists after cleanup is a bit strange. A new istio-cleanup.sh perhaps?